cfgov/preprocessed/css/molecules/form-field-with-button.less

```diff
@@ -0,0 +1,60 @@
+/* topdoc
+  name: Form Field with Button
+  family: cfgov-molecules
+  patterns:
+    - name: Default example
+      markup: |
+		<div class="m-form-field-with-button">
+		    <div class="form-group">
+		        <label for="name">
+		            <b></b>(required)
```

Comment:
Running it locally and seeing you're right, it's coming from somewhere. Leave it as is and we can discuss the usage in an issue.